src/types/message/message.proto.js

```diff
@@ -5,11 +5,18 @@
 module.exports = `
   message Message {
     message Wantlist {
+      enum WantType {
+        Block = 0;
+        Have = 1;
+      }
+
       message Entry {
         // changed from string to bytes, it makes a difference in JavaScript
         optional bytes block = 1;      // the block cid (cidV0 in bitswap 1.0.0, cidV1 in bitswap 1.1.0)
         optional int32 priority = 2;    // the priority (normalized). default to 1
         optional bool cancel = 3;       // whether this revokes an entry
+        WantType want_type = 4;         // Note: defaults to enum 0, ie Block
+        bool send_dont_have = 5;        // Note: defaults to false
```

Comment:
Is there _any_ way we can have `wantType`, `sendDontHave` etc.? 🙏 Snake case is not idiomatic JS and we already have to interact with Pascal case because of golang. It would be good not to add another! 😇

Reply:
hmm yeah I'm not sure why I made those snakey in the first place 🐍

Reply:
FWIW the protocol buffers style guide recommends snake_case for property names and CamelCase for message names.

https://developers.google.com/protocol-buffers/docs/style